packages/rs-dpp/src/state_transition/abstract_state_transition.rs

```diff
@@ -156,6 +159,9 @@ pub trait StateTransitionLike:
 
         bls.verify_signature(self.get_signature(), &data, public_key)
             .map(|_| ())
+            .or(Err(ProtocolError::from(ConsensusError::SignatureError(
```

Comment:
Please review this addition. 
I made it because signature verification error was ending up in `Result:Err` variant instead of being added to ValidationResult.

Reply:
Well, it should return an error here, shouldn't it? Or do you mean that it was returning just a generic error instead of `InvalidStateTransitionSignatureError`?

Reply:
Yes, it was a generic error